exercises/practice/isogram/.meta/config.json

```
@@ -1,6 +1,19 @@
 {
   "blurb": "Determine if a word or phrase is an isogram.",
   "authors": [],
```

Considering this: https://github.com/exercism/kotlin/commit/2f0259123a2e1355972b5cc8a98714e327258ab8#diff-4b8159e1fa13a3ed97750c1d3ec28a55765572b24b190544d28b46540267a657

```suggestion
  "authors": [
    "gandrianakis"
  ],
```

---

I can rebase and apply this suggestion by myself. But what should we do with failing checks? Should I ignore them? @ErikSchierboom please advice.

---

@dector I would probably commit the suggestion yourself and then squash on merge. Rebase is also fine with me.

Failing checks can be ignored in this PR. They can (and probably should) be fixed in separate PRs.